### ISSUE TITLE
Phosphene

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10591]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
A phosphene is the perception of light without light actually entering the eye, for instance caused by pressure applied to the closed eyes.

Cambridge dictionary : A phosphene is a phenomenon characterized by the experience of seeing light without light actually entering the eye.

PMID: 30306317 During a hydropic vertigo attack, the patients report nausea, vomiting, sweating, urge to defecate, urge to urinate, **phosphenes**, headache, photophobia, phonophobia and even transient loss of consciousness. A third of the patients does not experience auditory symptoms during the vertigo attacks.

**Parent term (use hpo.jax.org/app)**
 Entoptic phenomenon (see issue #10588 )

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
AB31.0 Meniere disease - ICD-11

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
added as HP:0020332
